added fav and title

kshepetska opened the pull request:
![Monosnap Nice gadgets - Google Chrome 2024-04-23 1](https://github.com/fs-jan24-codeup/phone-catalog/assets/155641185/8faf81bd-4e0c-4508-9ea7-9ece6782545f)

Vasya564 reviewed (changes requested):
I don't see which files are changed
Also we don't need these pictures of categories because we already have them

kshepetska commented:
> I don't see which files are changed Also we don't need these pictures of categories because we already have them

I don't know why these pictures are visible in changed files, cause I just deleted vite.svg and added favicon.svg
